build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -315,7 +332,7 @@ function setup_venv() {
       ;;
     stable)
       echo ":::: Using stable dependencies"
-      python3 -m pip install --no-cache-dir torch torchvision
+      python3 -m pip install --no-cache-dir torch==2.0.1 torchvision==0.15.2 --index-url https://download.pytorch.org/whl/cpu
```

Comment:
Why did you need to add version numbers here?
If we drop the version numbers here, would it automatically test with the latest stable version?

Reply:
I would leave it unpinned testing on latest stable. (Future planned work may take a while so meanwhile let it test on latest stable than being pinned to 2.0.1).